config/initializers/01_proc.rb

```diff
@@ -1,21 +0,0 @@
-# frozen_string_literal: true
-
-# www.mattsears.com/articles/2011/11/27ruby-blocks-as-dynamic-callbacks
-class Proc
-  def callback(callable, *args)
-    self === Class.new do
-      method_name = callable.to_sym
-      define_method(method_name) { |&block| block ? block.call(*args) : true }
-      define_method("#{method_name}?") { true }
-      def method_missing(*_args)
-        super
-        false
-      end
-
-      def respond_to_missing?(*)
-        super
-        true
-      end
-    end.new
-  end
-end
```

Comment:
I remember thinking this was a native feature I'd not seen before when I first came across it in the codebase.